README.md

```diff
@@ -45,6 +45,7 @@ Using fluent-plugin-logdna, you can send the logs you collect with Fluentd to Lo
   tags web,dev
   request_timeout 30000 ms
   ingester_domain https://logs.logdna.com
+  ingester_endpoint /logs/agent
```

Comment:
Believe this should be `/logs/ingest` shouldn't it?

Reply:
@matt-march, by default, it's `/logs/ingest` as documented - in this one, I'm just showing another example